C-Plus-Plus/other/Stable_Matching.cpp

```diff
@@ -0,0 +1,111 @@
+#include <iostream> 
```

Comment:
Add problem description in commented form. Prefer using #include <bits/stdc++.h> instead of adding all the headers.

Reply:
Done sir.